docs/syntax.rst

```diff
@@ -2,9 +2,14 @@
 The Guide To The FTL Syntax
 ===========================
 
-FTL is an open source, localization-specific scripting language used to process 
-gender, plurals, conjugations, and most of the other quirky elements of natural 
-language.
+FTL is a localization language used for describing translation resources.
```

Comment:
I prefer what I wrote over this. I took it from Wikipedia articles for HTML and CSS and I believe telling people that it's a language is more appropriate than that it's a syntax.